pyrk/test_logger.py

```diff
@@ -0,0 +1,71 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
I don't understand the purpose of this test_logger file which seems to be a semi-copy of the logger file and executes no tests.